src/pages/AddItem.jsx

```diff
@@ -1,6 +1,29 @@
-// 상품 등록하는 화면입니다.
+import { useEffect, useState } from "react";
+import ImageUpload from "../components/ImageUpload";
+import FormInput from "../components/FormInput";
+import "./AddItem.css";
+
 const AddItem = () => {
-  return <div>AddItem</div>;
+  const [isFormValid, setIsFormValid] = useState(false);
+
+  return (
+    <div className="additem">
+      <div className="additem-header">
+        <h2>상품 등록하기</h2>
+        <button className="submit-button" disabled={!isFormValid}>
+          등록
+        </button>
+      </div>
+
+      <div className="additem-image-section">
+        <ImageUpload />
+      </div>
+
+      <div className="additem-form-section">
+        <FormInput onFormValidChange={setIsFormValid} />
+      </div>
+    </div>
```

Comment:
💊 제안
input들이 모여있는 것이니 form으로 태그 작성하시면 좋겠습니다.
form의 동작상 이점을 위해 아래처럼 submit 버튼을 포함하는 영역을 form으로 변경해보세요.

```suggestion
    <form className="additem"> // 이렇게하면 input에서 submit 이벤트 발생시 submit 이벤트가 실행됩니다.
      <div className="additem-header">
        <h2>상품 등록하기</h2>
        <button type="submit" className="submit-button" disabled={!isFormValid} onSubmit={onSubmit}>
          등록
        </button>
      </div>

      <div className="additem-image-section">
        <ImageUpload />
      </div>

      <div className="additem-form-section">
        <FormInput onFormValidChange={setIsFormValid} />
      </div>
    </form>
```